pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -379,7 +379,8 @@ func getSupportedUnifiedControllers() sets.String {
 
 func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcontainerconfigs.Resources {
 	resources := &libcontainerconfigs.Resources{
-		SkipDevices: true,
+		SkipDevices:     true,
+		SkipFreezeOnSet: true,
```

Comment:
we also set up `libcontainerconfigs.Resources` and set `SkipDevices: true` in these places:

https://github.com/kubernetes/kubernetes/blob/b40d9e8dd5de13a11eb5c56c8fdb7794047764d9/pkg/kubelet/cm/container_manager_linux.go#L397-L399

https://github.com/kubernetes/kubernetes/blob/b40d9e8dd5de13a11eb5c56c8fdb7794047764d9/pkg/kubelet/dockershim/cm/container_manager_linux.go#L125-L129


should those set this new option as well?

Reply:
No (I should have explained it earlier but didn't know where to).

The reason is, `SkipFreezeOnSet` only makes sense for systemd v1 manager, and is ignored otherwise. In these two places you quoted we use fs[2] manager, i.e. no systemd involved.

In other words, it wouldn't hurt to set it, but it's useless (and may be misleading for those who read the code later).